packages/@lwc/engine-core/src/framework/vnodes.ts

```diff
@@ -74,8 +74,10 @@ export interface VFragment extends BaseVNode, BaseVParent {
 
     // which diffing strategy to use.
     stable: 0 | 1;
-    leading: VText;
-    trailing: VText;
+    // The leading and trailing nodes are text nodes <= v59 and comment nodes
+    // in v60+.
```

Comment:
This makes me sad :(. Especially if we get rid of these delimiters in the next major version.

If we move to using APIFeature rather than checking the version directly, this comment should be updated to reflect the name of the APIFeature being checked.